src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -21,16 +21,17 @@ type FSharpCompilerServiceChecker(hasAnalyzers, typecheckCacheSize) =
   let checker =
     FSharpChecker.Create(
       projectCacheSize = 200,
-      keepAllBackgroundResolutions = true,
       keepAssemblyContents = hasAnalyzers,
+      keepAllBackgroundResolutions = true,
       suggestNamesForErrors = true,
-      enablePartialTypeChecking = not hasAnalyzers,
+      keepAllBackgroundSymbolUses = true,
       enableBackgroundItemKeyStoreAndSemanticClassification = true,
-      keepAllBackgroundSymbolUses = true
+      enablePartialTypeChecking = not hasAnalyzers,
+      parallelReferenceResolution = true,
```

Comment:
We're enabling this in Ionide via a config flag using an envvar.  Could pass this along instead https://github.com/ionide/ionide-vscode-fsharp/blob/main/src/Core/LanguageService.fs#L714

Reply:
I guess tangentially, how much of this stuff should we expose via config?

Reply:
We can progressively add it where there's no clear answer so that users can unblock themselves/try things out. I'll add it here for example. Something like `enablePartialTypeChecking` though has rules that we can infer/follow - e.g. any analyzers auto-disable the feature.